cdc/sink/producer/kafka/metrics.go

```diff
@@ -213,14 +217,28 @@ func getBrokerMetricName(prefix, brokerID string) string {
 	return prefix + brokerID
 }
 
-func (sm *saramaMetricsMonitor) collectBrokerMetrics() error {
+func (sm *saramaMetricsMonitor) collectBrokers() {
+	start := time.Now()
 	brokers, _, err := sm.admin.DescribeCluster()
 	if err != nil {
-		return err
+		log.Warn("kafka cluster unreachable, "+
+			"use historical brokers to collect kafka broker level metrics",
+			zap.String("changefeed", sm.changefeedID),
+			zap.Any("role", sm.role),
+			zap.Duration("duration", time.Since(start)))
+		return
 	}
 
 	for _, b := range brokers {
-		brokerID := strconv.Itoa(int(b.ID()))
+		sm.brokers[b.ID()] = struct{}{}
```

Comment:
I think an int32 slice is more suitable here.

Reply:
we maintain brokers by set, it's easier for insertion.